mplc/docs/documentation.md

```diff
@@ -296,9 +297,73 @@ All methods available are:
 - "AIS_Kriging_S"
 - "SMCS"
 - "WR_SMC"
+- "Federated SBS linear"
+- "Federated SBS quadratic"
+- "Federated SBS constant"
 ```
 
-See below section [Contributivity measurement approaches studied and implemented](#contributivity-measurement-approaches-studied-and-implemented) for explanation of the different methods.  
+
+The methods are detailed below: 
+
+- `["Independent scores"]` **Performance scores** of models trained independently on each partner
+
+- [**Shapley values**](https://arxiv.org/pdf/1902.10275.pdf):  
```

Comment:
It looks weird that some title have an hyperlink and some don't. Maybe something like:
[**Shapley values**] - [paper](https://arxiv.org/pdf/1902.10275.pdf)

What do you think ?

Reply:
We cannot provide paper or resources for every method. ( step by step, pvrl, or Lflip for instance).
What about an hyperlink [resources](somelink,paper, or whatever useful) at the end of each paragraph ?

Reply:
Yes good idea !